### PR TITLE
Day 1

### PR DESCRIPTION
**Pull request by Myrdden**
- Completes first user story with expectation (added to board) of changes to authentication
  - Adds Github API Service and Github Repository model
  - Lists five repositories on User show page

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -367,6 +383,3 @@ DEPENDENCIES
 
 RUBY VERSION
    ruby 2.4.1p111
-
-BUNDLED WITH
```
Comment (Myrdden) [inline Gemfile.lock:371]: Apparently breaks Heroku deployment

File: app/facades/github_facade.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class GithubFacade
+  attr_reader :render
+  def initialize(user)
+    if user.token != nil
+      @render = "partials/github_repos"
+    else
+      @render = "partials/github_auth_prompt"
```
Comment (Myrdden) [inline app/facades/github_facade.rb:7]: TO BE ADDED WITH OAUTH

File: app/services/github_service.rb [hunk 3]
```diff
@@ -1,15 +1,13 @@
-# frozen_string_literal: true
-
 class GithubService
   def repo_info
-    params = { part: 'owner', id: id }
-    JSON.parse(conn.get('/user/repos', params).body, symbolize_names: true)
+    JSON.parse(conn.get('/user/repos').body, symbolize_names: true)
```
Comment (kylecornelissen) [inline app/services/github_service.rb:3]: Why did you take out the params?

Comment (Myrdden) [reply app/services/github_service.rb:3]: We weren't actually passing any params in so the variable wasn't doing anything

**Review by kylecornelissen — APPROVED** [review]
- The partials are a good way to save space in the users show page
- I like how you added the argument for the repos method in the GithubFacade class
- New user added, Doug

**Comment by kylecornelissen** [comment]
Let's write some tests before we merge

**Comment by kylecornelissen** [comment]
Added tests for github repo view

**Pull request merged**